lib/quill_delta.dart

```diff
@@ -474,7 +474,7 @@ class Delta {
   /// Returns a [Delta] containing differences between 2 [Delta]s
   ///
   /// Useful when one wishes to display difference between 2 documents
-  Delta diff(Delta other) {
+  Delta diff(Delta other, {bool cleanupSemantic = true}) {
```

Comment:
Could you add some docs regarding `cleanupSemantic`? It's not clear for me what semantic means in this context...